### DISCUSSION TITLE
Inconsistency in fields.json and NTD for notice-type CEI according to field OPA-36-Lot-Number

### DISCUSSION THREAD

**Discussion by gabrieledall** [OP-TED/eForms-SDK#450] [Q&A] [answered]
The field named "OPA-36-Lot-Number" is declared to be mandatory in fields.json but the Notice Type Definition for CEI does not contain any field with that name - in fact no NTD at all does contain a field with that name. Is that field not used any more?

Since CEI is not an eforms- noticetype, this inconsistency shoud not have effect on our work - but currently it does. We've found a workaround (adding this field to the CEI-NTD which we will never use). But we decided to let you know, anyway.

**Comment by bertrand-lorentz** [comment, marked as answer]
Thanks for your feedback !
This field was added only to be able to address the numerical value in BT-36-Lot, without the unit. It is only useful for display or data lookup. It will always have a value when BT-36-Lot has a value, so it shouldn't be indicated as mandatory, as it is indeed confusing.
We will remove the corresponding mandatory constraints.
